.appveyor.yml

```diff
@@ -10,17 +14,24 @@ cache:
 
 environment:
   matrix:
+    - nodejs_version: '10'
     - nodejs_version: '8'
     - nodejs_version: '6'
 
+matrix:
+  fast_finish: true
+
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
   - yarn install
 
+build: off
```

Comment:
According to the [Appveyor reference](https://www.appveyor.com/docs/appveyor-yml/), this will "disable automatic tests". I am not clear on why you are adding this here. Could you please clarify?

Reply:
No it doesn't. It's off because we're not building a MS solution.

Reply:
![image](https://user-images.githubusercontent.com/22251956/40291361-36d26c48-5ce2-11e8-8e01-e17b184cedd4.png)

Reply:
Yeah, but like I said -- we're not running MS build. We have no solution file. So it's off.